src/components/OmouComponents/ReadMore.js

```diff
@@ -0,0 +1,31 @@
+import React,{useState} from 'react';
+import PropTypes from 'prop-types';
+import Typography from '@material-ui/core/Typography'
+import theme from "../../theme/muiTheme";
+
+const ReadMore = (props) => {
+
+    const [readMore, setReadMore] = useState(false);
+    const handleReadMoreClick = () => setReadMore(!readMore);
+
+    return (
+        <Typography variant="body1" align="left" style={{ wordBreak: "break-word" }}>
+            {
+                !readMore && props.body.length > 110 ? props.body.substring(0,60) + "..." : props.body
+            }
+            {props.body.length > 110 && (
```

Comment:
since we're changing how the component is used, refactor `props.body.length` to `props.children.length` for reference: https://reactjs.org/docs/composition-vs-inheritance.html